src/framework/core/js/ModelTransformation.js

```diff
@@ -382,6 +383,11 @@ var fluid = fluid || fluid_2_0_0;
     };
     // unsupported, NON-API function
     fluid.model.transform.handleInvertStrategy = function (transformSpec, transform, transformOpts) {
+        transformSpec = fluid.copy(transformSpec);
```

Comment:
are you sure your diff is up to date? I've deleted the old fluid.copy from invertPaths on the old line 164